data/query.go

```diff
@@ -26,33 +28,34 @@ func ReviewQuery(ctx context.Context, cfg *config.Config, urlQuery url.Values, c
 	var validatedQueryParams SearchURLParams
 	validatedQueryParams.Query = urlQuery.Get("q")
 
-	err := reviewPagination(ctx, cfg, urlQuery, &validatedQueryParams)
-	if err != nil {
-		log.Error(ctx, "unable to review pagination", err)
-		return validatedQueryParams, err
+	paginationErr := reviewPagination(ctx, cfg, urlQuery, &validatedQueryParams)
+	if paginationErr != nil {
+		log.Error(ctx, "unable to review pagination", paginationErr)
+		return validatedQueryParams, paginationErr
 	}
 
 	reviewSort(ctx, cfg, urlQuery, &validatedQueryParams)
 
-	err = reviewFilters(ctx, urlQuery, &validatedQueryParams)
-	if err != nil {
-		log.Error(ctx, "unable to review filters", err)
-		return validatedQueryParams, err
+	contentTypeFilterError := reviewFilters(ctx, urlQuery, &validatedQueryParams)
+	if contentTypeFilterError != nil {
+		log.Error(ctx, "bad content type filters set", contentTypeFilterError)
```

Comment:
Perhaps a bit nit-picky but maybe "incorrect" or "invalid" would be better here? Up to you.